//2024. Caleb Papay

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

message skelEntry_proto
{
	optional sint32   nodeIndex = 1;
	optional sint32   parentIndex = 2;
	optional bytes    pNodeName = 3;

}

// -- End of File ---
